Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -304,14 +303,39 @@
     - type: StorageFill
       contents:
         - id: BoxSurvival
-        - id: WeaponPulsePistol
-        - id: MedkitFilled
+        - id: LightReplacer
         - id: BoxLightMixed
         - id: BoxLightMixed
         - id: Soap
         - id: CrowbarRed
         - id: AdvMopItem
 
+# Death Squad
+
+- type: entity
+  noSpawn: false
+  parent: ClothingBackpackERTSecurity
+  id: ClothingBackpackDeathSquadFilled
+  name: death squad backpack
+  description: Holds the kit of CentComm's most feared agents.
+  components:
+    - type: StorageFill
+      contents:
+        - id: BoxSurvival
+        - id: WeaponPulseRifle
+        - id: WeaponPulsePistol
+        - id: WeaponRevolverMateba
+        - id: SpeedLoaderMagnumAP
+        - id: SpeedLoaderMagnumAP
+        - id: BoxFlashbang
+        - id: ToolDebug # spanish army knife
+        - id: WelderExperimental
+        - id: Hypospray
+        - id: MicroBombImplanter # crew will try to steal their amazing hardsuits
+        - id: FreedomImplanter
```

Comment:
Mind shield implanter is useful too, as well as for any other ERT roles, unless they start off mindshielded by default.